zrml/swaps/src/migrations.rs

```diff
@@ -1,2 +1 @@
-pub mod _0_1_1_add_pool_status;
-pub mod _0_1_2_add_pool_market_id;
```

Comment:
Ppl on https://docs.zeitgeist.pm/battery-park#get-connected are still using the same initial fb127223ea8990bb27819dbbb9b15a46d7ffea73 build and no new release was published since then. Therefore, these two migrations are now merged into one.